packages/utils/src/cjs.js

```diff
@@ -1,6 +1,10 @@
+export function isExternalDependency (id) {
+  return id.includes('/node_modules/')
+}
+
 export function clearRequireCache (id) {
   const entry = require.cache[id]
-  if (!entry || id.includes('node_modules')) {
+  if (!entry || isExternalDependency(id)) {
```

Comment:
not sure if it was deliberate to use `node_modules` without separators here?

Reply:
Indeed we can use path sep (don't forgot to include both \\ and / in regex ;))